packages/keystone/fetch.js

```diff
@@ -1,14 +1,26 @@
+const os = require('os')
+
+const { pickBy } = require('lodash')
 const nodeFetch = require('node-fetch')
 
+const conf = require('@open-condo/config')
+
 const { getExecutionContext } = require('./executionContext')
 const { getLogger } = require('./logging')
 const Mertrics = require('./metrics')
 
+
 const logger = getLogger('fetch')
 
 const FETCH_COUNT_METRIC_NAME = 'fetch.count'
 const FETCH_TIME_METRIC_NAME = 'fetch.time'
 
+const HOSTNAME = os.hostname()
+const NAMESPACE = conf.NAMESPACE
+const DEPLOYMENT = (HOSTNAME.startsWith('condo') &&
+    HOSTNAME.split('-').length > 2 &&
+    HOSTNAME.split('-').slice(0, 3).join('-')) || HOSTNAME
```

Comment:
Maybe it would be better to use a regular expression here?

Reply:
Rewrote this to a function to make it more readable